sdk/confidentialledger/ci.yml

```diff
@@ -0,0 +1,35 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+    - master
+    - main
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/confidentialledger/
```

Comment:
Should we include the `ci.yaml` file here as well?

Reply:
Disregard;  this is my error in reading comprehension.

Reply:
this is the top level path.